server/index.js

```diff
@@ -13,8 +13,10 @@ app.use(bodyParser.urlencoded({
 app.use(bodyParser.json())
 
 
-app.get('/media', (req,res) =>{
-  db.getData((data) => {
+app.get('/media/:id', (req,res) =>{
+  var id = req.params.id;
+  db.getData(id, (data) => {
+    console.log(data);
```

Comment:
Is there a way you could send the appropriate status code back as well as the data?

Reply:
I agree with @derrickdbrandon - I think it'd be a good idea to send back a status code